src/sentry/conf/server.py

```diff
@@ -915,6 +915,9 @@ def create_partitioned_queues(name):
 # over SENTRY_FRONTEND_PROJECT or SENTRY_PROJECT
 SENTRY_FRONTEND_DSN = None
 
+# Configuration for JavaScript's whitelistUrls - defaults to ALLOWED_HOSTS
+SENTRY_FRONTEND_WHITELIST_URLS = None
```

Comment:
Open to other ideas here, but i dont think ALLOWED_HOSTS can be relied on for our production usage (as we bind some internal domains which AFAICT wouldnt make sense). Might be worth chatting live/Slack tho to see if that is true..

Current sentry prod whitelist:

```
whitelistUrls: ['sentry.io', 's1.sentry-cdn.com', 'dev.getsentry.net', '<anonymous>'],
```